### PR TITLE
Remove comment

### PR DESCRIPTION
**Pull request by Chiragmodi1247**
*Issue #340 , if available:*

*Description of changes:*
Removed weird comments

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by Chiragmodi1247** [comment]
I have done two separate PRs. One to convert all forin to forEach and another one to remove the weird comments mentioned in one of your issues. Is there anything wrong?

**Comment by dbanksdesign** [comment]
This PR contains more changes than just removing the comment. Can you remove all other changes? Thanks!

**Comment by chazzmoney** [comment]
As Danny mentioned, you have more changes beyond removing the comment in this PR.

1. You have [two commits](https://github.com/amzn/style-dictionary/pull/343/commits), of which I think you only wanted [this one](https://github.com/amzn/style-dictionary/pull/343/commits/6df090f5d0924ecc2f45be7c48a1f71bef8a7869). You need to remove [this commit](https://github.com/amzn/style-dictionary/pull/343/commits/3e72fa6a7b04f9cbf15f196d8cddf223f5589eee).
2. As per the link above to the remove comments commit, there are additional changes you have made, which I can comment on directly to help you remove them.

File: __tests__/__output/json-nested.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "base": {
+      "red": {
+        "primary": "#611D1C",
+        "secondary": {
+          "inverse": "#000000"
+        }
+      }
+    }
+  }
+}
```
Comment (chazzmoney) [inline __tests__/__output/json-nested.json]: This change should be removed.

File: lib/register/action.js [hunk 2]
```diff
@@ -10,7 +10,7 @@
  * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
  * and limitations under the License.
  */
-var chalk = require('chalk');
+// var chalk = require("chalk");
```
Comment (chazzmoney) [inline lib/register/action.js]: This change should be removed.

**Comment by chazzmoney** [comment]
We just merged a large PR with documentation edits. @Chiragmodi1247 could you rebase and fix any conflicts please. Thank you!

**Comment by dbanksdesign** [comment]
@Chiragmodi1247 any update on this? Let me know if you need any help!

**Comment by dbanksdesign** [comment]
Closing this as there is no recent activity and it is not a critical change.

**Pull request closed without merging**